### ISSUE TITLE
Be able to store feature file in sub folders

### ISSUE THREAD

**Issue by FlorianFlouquet** [PiotrFLEURY/pickled_cucumber#30]
**Is your feature request related to a problem? Please describe.**
As of today, all the feature files must be in the same folder, otherwise we can't generate .cucumber files.

Example:
```
/test
      /features
            /file1.feature
            /file2.feature
            /file3.feature
```

**Describe the solution you'd like**
Be able to store feature files in different folders and sub folders, as follows:

```
/test
      /features
            /context1
            /context2
                  /context2-A
                        /file1.feature
                  /context2-B
                        /file2.feature
                        /file3.feature
```

**Comment by FlorianFlouquet** [comment]
A [pull request](https://github.com/PiotrFLEURY/pickled_cucumber/pull/31) has been opened to solve this issue

**Comment by PiotrFLEURY** [comment]
Fixed by #31
